service/matching/handler/engine_integration_test.go

```diff
@@ -740,24 +739,7 @@ func (s *matchingEngineSuite) ConcurrentAddAndPollTasks(taskType int, workerCoun
 	s.matchingEngine.config.RangeSize = rangeSize // override to low number for the test
 	s.matchingEngine.config.TaskDispatchRPSTTL = time.Nanosecond
 	s.matchingEngine.config.MinTaskThrottlingBurstSize = dynamicconfig.GetIntPropertyFilteredByTaskListInfo(_minBurst)
-
-	mgr, err := tasklist.NewManager(
```

Comment:
I see other test cases creating such side tasklist manager. If this is not needed (assuming `s.matchingEngine.getTaskListManager` is doing same thing) then we can update those cases too I guess